apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/FormSelectors/FromRow.tsx

```diff
@@ -188,10 +212,18 @@ function FromRow({
             unit={unit}
             // Flow complains if this prop is missing…
             renderRight={null}
-            error={fromAmountError}
           />
         </InputSection>
       </Box>
+
+      {(!!fromAmountError || !!fromAmountWarning) && (
+        <SwapStatusContainer isError={!!fromAmountError}>
+          <WarningSolidMedium size={16} />
```

Comment:
With any kind of error are we going to show this icon? event with fromAmountError?